probable_immunity_web_app/static/style.css

```diff
@@ -0,0 +1,3 @@
+p.back {
+    text-align: right;

```

Comment:
If you look at the CI checks, Codacy is complaining that the indentation here is 4 spaces instead of two (which is the convention AFAICT), can you fix that?

Reply:
Sure. I will do.